doc/source/reference/random/generator.rst

```diff
@@ -71,17 +71,17 @@ By default, `Generator.permuted` returns a copy.  To operate in-place with
 `Generator.permuted`, pass the same array as the first argument *and* as
 the value of the ``out`` parameter.  For example,
 
-    >>> rng = np.random.default_rng()
+    >>> rng = np.random.default_rng(12345)
```

Comment:
This is still incorrect, can you revert this?

Reply:
Done. I reverted all such changes and used SKIP to avoid erroring on the random output.